src/seedu/addressbook/data/person/Address.java

```diff
@@ -11,7 +11,13 @@
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
-
+    
+    
+    private Block block = new Block();
```

Comment:
Is it necessary to initialize the variables at this stage? We can do it when we get the information